CodeSniffer/Standards/Generic/Sniffs/Commenting/FixmeSniff.php

```diff
@@ -72,7 +72,7 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
             // Clear whitespace and some common characters not required at
             // the end of a fixme message to make the error more informative.
             $type         = 'CommentFound';
-            $fixmeMessage = trim($matches[1]);
+            $fixmeMessage = trim(@$matches[1]);
```

Comment:
Hiding errors is bad practice especially within a library, that enforces good practice.

I recommend doing proper array key existence check.